### PR TITLE
Password reset

### PR DESCRIPTION
**Pull request by EvonneNgXinYi**
Build up password reset system

### CODE REVIEW

**Comment by EvonneNgXinYi** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
Your PR is having 2 commits, merge to 1 commit

**Comment by EvonneNgXinYi** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -22,7 +22,6 @@ gem "rails-i18n"
 gem "sass-rails", ">= 6"
 gem "turbolinks", "~> 5"
 gem "webpacker", "~> 4.0"
-gem "kaminari"
```
Comment (chan227) [inline Gemfile:25]: we accidentally wrote two same of coding of this, so we delete one to prevent any confuse.

**Pull request merged**